app/controllers/index.rb

```diff
@@ -3,22 +3,29 @@
 end
 
 get '/todos' do
-  p "SHIT"
   @todos = Todo.all
   erb :index
 end
 
 post '/todos' do
   if request.xhr?
-    p "working"
-    Todo.create(name: params[:name])
+    new_todo = Todo.create(name: params[:name])
+    erb :_delete, locals: {todo: new_todo}, layout: false
   else
-    p "breaking"
     redirect '/todos'
   end
 end
 
 
-get '/tasks/new' do
+get '/todos/new' do
   erb :_form, layout: false
 end
+
+delete '/todos/:id' do
+  if request.xhr?
+    Todo.find(params[:id]).destroy
+    return params[:id]
+  else
```

Comment:
response can be either a string(text/html), json, xml
it cannot be an integer because integers are treated as status code.. 200 is a valid status code